app/src/main/res/color/tab_icon_color_selector.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
   <item android:color="@color/white" android:state_selected="true"/>
-  <item android:color="@color/white_70"/>
+  <item android:color="@color/white_80"/>
```

Comment:
Why is this change part of this PR?

Reply:
The Issue -1 of #2295 says the inactive Tab icons have wrong contrast ratio, so Chantel suggested that we should use this particular color for inactive tab icons. Hence I tweaked the background drawable of Tab Icons

Reply:
Okay got it. Thanks.